bin/migrateDirtyDBtoRealDB.js

```diff
@@ -36,7 +31,7 @@ const util = require('util');
   await db.init();
 
   console.log('Waiting for dirtyDB to parse its file.');
-  const dirty = dirtyDb('var/dirty.db');
+  const dirty = dirtyDb(`${__dirname}/../var/dirty.db`);
```

Comment:
This is going to have to change with PR #4707.

Reply:
good point, easily done then tho :)